deepspeed/accelerator/cuda_accelerator.py

```diff
@@ -110,6 +110,13 @@ def max_memory_reserved(self, device_index=None):
     def total_memory(self, device_index=None):
         return torch.cuda.get_device_properties(device_index).total_memory
 
+    # Data types
+    def is_bf16_supported(self):
+        return torch.cuda.is_bf16_supported()
+
+    def is_fp16_supported(self):
```

Comment:
pytorch have torch.cuda.is_bf16_supported() but don't have is_fp16_supported(), check HW version is still the only way to tell whether CUDA device support fp16.

Reply:
Good catch!